File: ED/practical-classes/Ficha3/Ex3/src/pt/isec/forgotten/figures/Rectangle.java
package pt.isec.forgotten.figures;

public class Rectangle extends Figure {
    private final float height;
    private final float width;

    public Rectangle(float height, float width) {
        this.height = height;
        this.width = width;
    }

    @Override
    public float getArea() {
        return height * width;
    }

    @Override
    public float getPerimeter() {
        return (height * 2) + (width * 2);
    }
}

File: PWEB/practical-classes/Ficha1/Ex3/Ex3/Rectangle.cs
namespace Ex3 {
    public class Rectangle : IGeometricShape {
        private double _length;
        private double _width;

        public Rectangle(double length, double width) {
            _length = length;
            _width = width;
        }

        public double GetArea() => _length * _width;
        public double GetPerimeter() => (_length * 2) + (_width * 2);

        public override string ToString() {
            return "Rectangle " + _width + "x" + _length + '\n'
                   + "Area: " + GetArea() + " Perimeter: " + GetPerimeter();
        }
    }
}